react/features/base/media/middleware.any.ts

```diff
@@ -163,7 +165,8 @@ MiddlewareRegistry.register(store => next => action => {
 function _appStateChanged({ dispatch, getState }: IStore, next: Function, action: AnyAction) {
     if (navigator.product === 'ReactNative') {
         const { appState } = action;
-        const mute = appState !== 'active' && !isLocalVideoTrackDesktop(getState());
+        const bgVideoActiveEnabled = Boolean(getFeatureFlag(getState(), BG_VIDEO_ACTIVE_ENABLE));
+        const mute = appState !== 'active' && !isLocalVideoTrackDesktop(getState()) && !bgVideoActiveEnabled;
```

Comment:
Should this line be bgVideoActiveEnabled, instead of !bgVideoActiveEnabled? Because the appSate is !==

Reply:
The conditions for muting the video are:
    'muted == true'
     The following three conditions are met simultaneously:
     1.Non-active state
     2.There is a shared video track on the local desktop, and the track is not muted.
     3.Background video active is set to false.
      So:
      If we set bgVideoActiveEnabled to true, mute == false, the video will not be muted.

Reply:
But if iOS wants to make the video stream normal in background mode, it must implement VOIP or Picture in Picture (PIP).

VOIP is not allowed in the Chinese market
PIP webRtc does not currently support iOS and is waiting for PRs from other developers.
https://github.com/react-native-webrtc/react-native-webrtc/pull/1619